core/templates/core/reported-post.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block extra_css %}
+<link rel="stylesheet" href="{% static 'core/css/style.css' %}">
+{% endblock %}
+
+{% block content %}
+    <div class="container mt-5">
+        <h1 class="mb-4">Reported Post</h1>
+        <div class="card bg-dark shadow-sm mb-3 text-light">
+            <div class="card-header">
+                <div class="d-flex justify-content-between">
+                    <div>
+                        <a href="#" class="link-primary">{{ report.post.title }}</a>
```

Comment:
Please review the whole PR and use real links not # everywhere. I will not add any more comments on #.